components/ep/layout/FooterContacts.vue

```diff
@@ -0,0 +1,70 @@
+<!--
+  ~ Copyright 2023 Exactpro (Exactpro Systems Limited)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<script setup lang="ts">
+const config = useAppConfig()
+let contacts = config.exactproDocs.contacts
+type Contact = (typeof config.exactproDocs.contacts)[number]
+
+if (contacts === undefined || contacts.length === 0) {
```

Comment:
This can be shortened.

`?.` returns `undefined` if `length` does not exist in the object (`length` does not exist in `undefined`).

`!undefined` = `true`
`!0` = `true`

```suggestion
if (!contacts?.length) {
```